sema4ai/package.json

```diff
@@ -227,7 +227,7 @@
             "panel": [
                 {
                     "id": "sema4ai-python-view-output",
-                    "title": "Robo Tasks Output",
+                    "title": "Task/Action Output",
```

Comment:
The `package.json` is generated by running `codegen/codegen_package.py`, so, you need to update the references inside the `codegen` dir and then rerun to update the `package.json`.

Reply:
oh yeah, good catch!